index.html

```diff
@@ -53,20 +55,20 @@ <h5 class="subline-image-big">ARTE</h5>
             <div class="row-image-1">
                   
                 <div class="column-image-1">
-                    <a href="https://www.s-f.family/about/" target="s-f"><img src="./assets/images/image_2.png" alt="SuF" class="image-small"></a>
+                    <a href="https://jenberlin.github.io/Quiz/" target="quiz"><img src="./assets/images/Quiz.png" alt="quiz" class="image-small"></a>
 
                         <div class="text-box-image-small">
-                            <h4 class="header-image-small">Agency</h4>
-                                <h5 class="subline-image-small">S&F</h5>
+                            <h4 class="header-image-small">Web Application</h4>
+                                <h5 class="subline-image-small">Coding Quiz</h5>
                         </div>
                 </div>
             
                 <div class="column-image-1">
-                    <a href="https://kiron.ngo/en/" target="kiron"><img src="./assets/images/image_3.png" alt="Kiron" class="image-small"></a>
+                    <a href="https://jenberlin.github.io/Weather-App/" target="Weather-App"><img src="./assets/images/Weather_App.png" alt="Weather-App" class="image-small"></a>
```

Comment:
great remembrance of the use of alt tags here, this is very important for accessibility!
Be careful with formatting your code, if you can, place the img tag on a separate line to the anchor tag for easier readability.